source/blog/2015-06-17-ember-data-1-13-released.md

```diff
@@ -0,0 +1,376 @@
+    @param {DS.Snapshot} snapshot
+    @return {Boolean}
+  */
+  shouldBackgroundReloadRecord: function(store, snapshot),
+```
+
+Symmetric methods have also been added for `store.findAll`.
+
+`Adapter#shouldReloadAll` is called by the store after `store.findAll`is called. If it returns true the store will reload the record from the backend and it will wait to resolve the promise until the reload is complete. `shouldReloadAll` is never checked if the record is not in the store or if the user passed `{ reload: true}` in the options argument to `findAll`. The default implementation for this method in Ember Data 1.13 is to always return `true`. However in Ember Data 2.0 this will be changed to return `true` when there are records in the store of the requested type.
+
+```js
+  /**
+    @method shouldReloadAll
+    @param {DS.Store} store
+    @param {DS.SnapshotRecordArray} snapshotRecordArray
+    @return {Boolean}
+  */
+  shouldReloadAll: function(store, snapshotRecordArray),
+```
+
+`Adapter#shouldBackgroundReloadAll` is called by the store when the store is resolving the `findAll` because `shouldReloadAll` returned true. If this method returns `true` the store will attempt to reload the record from the adapter in the background. The default implementation for this method in both Ember Data 1.13 and Ember Data 2.0 is to always return `true`.
+
+```js
+  /**
+    @method shouldBackgroundReloadAll
+    @param {DS.Store} store
+    @param {DS.SnapshotRecordArray} snapshotRecordArray
+    @return {Boolean}
+  */
+  shouldBackgroundReloadAll: function(store, snapshotRecordArray),
+```
+
+### JSON-API Support
+
+Ember Data 1.13 comes with support for [JSON-API](http://jsonapi.org/) 1.0. There is a new `DS.JSONAPIAdapter` and `DS.JSONAPISerializer` that support working with JSON-API backends. In Ember Data 2.0 the JSON-API adapter and serializer will be the default adapter loaded by Ember Data.
+
+### New Serializer API
+
+The internal data format used by Ember Data's `store.push` method and serializers has evolved over the 3.5 years of the project. Unfortunately, this has resulted in some strange gotcha's around use cases like meta data, polymorphic relationships and relationship links. Especially when these use features are combined together. Another problem with Ember Data's internal format has been its lack of documentation which has made it hard for to create custom serializers for Ember Data without already having a strong understanding of the Ember Data codebase.
+
+In order to solve these problems Ember Data's is fully embracing JSON-API. `store.push` now takes a JSON-API document and serializers are expected to return a JSON-API document when extracting the adapter response. JSON-API solves many of the gotchas Ember Data's internal format has struggles with over the years and it is a [well documented format](http://jsonapi.org/format/) that serializer authors can target without having to know about Ember Data's internals.
```

Comment:
Should we change "when extracting the adapter response" to "when normalizing the adapter response" since we've just renamed all extract* methods to normalize*Respone?